scripts/circle_deploy.sh

```diff
@@ -64,3 +56,21 @@ else
  echo "...publishing"
  npm publish
 fi
+
+#
+# gh-pages
+#
+echo "...deploying gh-pages"
+git push -f $CIRCLE_BRANCH:gh-pages
+git checkout gh-pages
+gulp docs
+git add .
+git add -f docs/build
+
+if [[ -n $(git status --porcelain) ]]; then
+  echo "...starting push, gh-pages is dirty after build"
+  git commit -n -m "deploy gh-pages by $CIRCLE_USERNAME [ci skip]"
```

Comment:
Rather than `[ci skip]` here it feels more straightforward to add a branch build specification to `circle.yml` and `ignore` the `gh-pages` branch: https://circleci.com/docs/configuration#branches